style.css

```diff
@@ -432,14 +441,86 @@
   cursor: not-allowed;
 }
 
-.footer {
-  color: #fff;
+/* The Modal (background) */
+.modal {
+  display: none; /* Hidden by default */
+  position: fixed; /* Stay in place */
+  z-index: 1; /* Sit on top */
+  background-image: url(./assets/ham_bot.svg);
+  background-repeat: no-repeat;
+  background-position: top right;
+  background-size: 30%;
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+
+  /* Fallback color */
+  background-color: #191675 !important;
+
+  /* Black w/ opacity */
+  background-color: rgba(0, 0, 0, 0.4);
+}
+
+/* Modal Content/Box */
+.modal-content {
+  background-color: rgba(255, 255, 255, 0.16);
+  margin: 51px 24px 151px 24px;
```

Comment:
- [ ] The popup window looks great 😄 but it still needs some minor modification to make your portfolio perfect 👍 
As you can see in the screenshot below, the pop window is not centered 👇 
![Screenshot from 2022-07-29 12-26-45](https://user-images.githubusercontent.com/87965534/181729521-6e9b1921-384a-46ed-98bc-ac89e9ad1d29.png)
This can be fixed by changing this line to `margin: 51px auto 151px` to center the popup horizontally and match the Figma design as required 👇 
![Screenshot from 2022-07-29 12-29-15](https://user-images.githubusercontent.com/87965534/181729816-f74357f8-f7ea-4a24-9987-9a475a1e98b3.png)

Reply:
@Sahar-AbdelSamad Consider it done.